keras/applications/resnet_rs.py

```diff
@@ -0,0 +1,957 @@
+    drop_connect_rate=0.2,
+    include_top=True,
+    block_args: List[Dict[str, int]] = None,
+    model_name="resnet-rs",
+    pooling=None,
+    weights="imagenet",
+    input_tensor=None,
+    classes=1000,
+    classifier_activation: Union[str, Callable] = "softmax",
+    include_preprocessing=True,
+):
+    """Build Resnet-RS model, given provided parameters.
+
+    Args:
+        depth: Depth of ResNet network.
+        input_shape: optional shape tuple. It should have exactly 3 inputs
+            channels, and width and height should be no smaller than 32.
+            E.g. (200, 200, 3) would be one valid value.
+        bn_momentum: Momentum parameter for Batch Normalization layers.
+        bn_epsilon: Epsilon parameter for Batch Normalization layers.
+        activation: activation function.
+        se_ratio: Squeeze and Excitation layer ratio.
+        dropout_rate: dropout rate before final classifier layer.
+        drop_connect_rate: dropout rate at skip connections.
+        include_top: whether to include the fully-connected layer at the top of the network.
+        block_args: list of dicts, parameters to construct block modules.
+        model_name: name of the model.
+        pooling: optional pooling mode for feature extraction when `include_top`
+            is `False`.
+            - `None` means that the output of the model will be
+                the 4D tensor output of the
+                last convolutional layer.
+            - `avg` means that global average pooling
+                will be applied to the output of the
+                last convolutional layer, and thus
+                the output of the model will be a 2D tensor.
+            - `max` means that global max pooling will
+                be applied.
+        weights: one of `None` (random initialization), `'imagenet'`
+            (pre-training on ImageNet), or the path to the weights file to be loaded.
+            Note: one model can have multiple imagenet variants depending on
+            input shape it was trained with. For input_shape 224x224 pass
+            `imagenet-i224` as argument. By default, highest input shape weights are
+            downloaded.
+        input_tensor: optional Keras tensor (i.e. output of `layers.Input()`) to
+            use as image input for the model.
+        classes: optional number of classes to classify images into, only to be
+            specified if `include_top` is True, and if no `weights` argument is
+            specified.
+        classifier_activation: A `str` or callable. The activation function to
+            use on the "top" layer. Ignored unless `include_top=True`. Set
+            `classifier_activation=None` to return the logits of the "top" layer.
+        include_preprocessing: Boolean, whether to include the preprocessing layer
+            (`Rescaling`) at the bottom of the network. Defaults to `True`.
+            Note: Input image is normalized by ImageNet mean and standard deviation.
```

Comment:
lets change "Note:" to "Note-".  "Note:" gets picked up as a parameter name by our docstring generator

Reply:
Done